doc/nrf/known_issues.rst

```diff
@@ -2302,12 +2339,12 @@ SoftDevice Controller
 
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
-.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0
+.. rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0
 
 DRGN-18568: When the :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option is used, the actual value of radio output power is lower than expected.
```

Comment:
@rugeGerritsen : DRGN-18568 has been fixed by https://github.com/nrfconnect/sdk-nrf/pull/10143 ?

Reply:
Yes

Reply:
So here,  v2-3-0 should be removed